crates/wasm/wit/main.wit

```diff
@@ -13,6 +13,7 @@ world bindings {
     import client-input: pkg.client-input
     import client-camera: pkg.client-camera
     import client-window: pkg.client-window
+    import client-experimental: pkg.client-experimental
```

Comment:
I'd suggest moving this to client-experimental-mesh; we're likely to have more experimental modules in the future

Reply:
Sure. I was thinking name spacing experimental modules under `client-experimental` module, but it `wit` probably doesn't support it?

Reply:
I'm not actually sure, give it a shot?

Reply:
I skimmed through https://github.com/WebAssembly/component-model/blob/main/design/mvp/WIT.md, but there doesn't seem to be a way to nest `world` or `interface`, and there are no mentions of modules.

Reply:
Let's go with `client-experimental-X`, where `x=mesh/texture/material`.